code/index.html

```diff
@@ -4,27 +4,243 @@
+    <link rel="stylesheet" href="./style.css" />
+    <link
+      href="//netdna.bootstrapcdn.com/font-awesome/3.2.1/css/font-awesome.css"
+      rel="stylesheet"
+    />
   </head>
+
   <body>
-    <!-- video or image as a header is cool :) -->
+    <header>
+      <div class="nav-container">
+        <a class="nav-container__item" href="#">Home</a>
+        <a class="nav-container__item" href="#">About</a>
+        <a class="nav-container__item" href="#">Contact</a>
+      </div>
+
+      <div class="hero">
+        <h1 class="hero__topic">Did you sleep well?</h1>
+
+        <div class="hero__video">
+          <video autoplay muted loop class="desktop">
+            <source
+              src="videos/SampleVideo_1280x720_1mb.mp4"
+              type="video/mp4"
+            />
+          </video>
+          <video autoplay muted loop class="tablet">
+            <source src="videos/SampleVideo_720x480_1mb.mp4" type="video/mp4" />
+          </video>
+          <video autoplay muted loop class="mobile">
+            <source src="videos/SampleVideo_360x240_1mb.mp4" type="video/mp4" />
+          </video>
+        </div>
+      </div>
+    </header>
+
+    <div class="main-content">
+      <!-- Signup form -->
+
+      <section class="main-content__section">
+        <form
+          class="signup-form"
+          action="http://httpbin.org/anything"
+          method="POST"
+        >
+          <div class="input-container">
+            <label for="name">Name:</label><br />
+            <input
+              class="input-container__input input-container__input--full-width"
+              type="text"
+              placeholder="Your Name"
+              id="name"
+              name="name"
+              required
+            />
+          </div>
+
+          <div class="input-container">
+            <label for="last-name">Last Name:</label><br />
+            <input
+              class="input-container__input input-container__input--full-width"
+              type="text"
+              placeholder="Your Last Name"
+              id="last-name"
+              name="last-name"
+              required
+            />
+          </div>
+
+          <div class="input-container">
+            <label for="email">Enter your email:</label><br />
+            <input
+              class="input-container__input input-container__input--full-width"
+              type="email"
+              placeholder="Your Email"
+              id="email"
+              name="email"
+              required
+            />
+          </div>
+
+          <div class="input-container">
+            <label for="password">Password:</label><br />
+            <input
+              class="input-container__input input-container__input--full-width"
+              type="password"
+              placeholder="Your Password"
+              id="password"
+              name="pasword"
+              required
+              maxlength="3"
```

Comment:
Super small thing, but let the attribute without value be the last one (so, `required`)